src/app/profiles/App/Profiles/Profile/withConnect.ts

```diff
@@ -54,17 +54,10 @@ const mergeProps = (
 ) => ({
   ...ownProps,
   ...stateProps,
+  ...dispatchProps,
   fetchedAll: stateProps.contributor
     ? stateProps.fetchedAll(stateProps.contributor?.id)
     : false,
-  subscribe: () => {
```

Comment:
Can you tell me what was not working here ? Or is it an approach matter ? 

Reply:
I think you previously changed this somewhere in the pagination, but I'm not sure you needed to, correct me if I'm wrong.
But the thing is it introduce a bug, since the subscribe method is also used on the "Similar profile component", all subscriptions done via this component were subscribing to the main informer instead ^ ^

Reply:
Yes, I get it, thanks